File: ChibuzorTasks/SumCalculator.java
import java.util.Scanner;

public class SumCalculator {
	public static void main(String[] arg) {
		Scanner input = new Scanner(System.in);
		String repeat;

		do {
			System.out.print("Please enter your first number:");
			double firstNumber = input.nextDouble();

			System.out.print("Please enter your second number:");
			double secondNumber = input.nextDouble();

			double sumOfNumbers = firstNumber + secondNumber;
			System.out.println("The sum is: " + sumOfNumbers);
			
			input.nextLine();
			System.out.print("Do you wish to perform the operation again? (yes/no)");
			repeat = input.nextLine();
			 
		} while (repeat.equals("yes"));
		System.out.println("Thank You");	
	}
}

File: ChibuzorTasks/sum_calculator.py
repeat = "yes"
while repeat == "yes":
	first_number = float(input("Enter the first number: "))
	second_number = float(input("Enter the second number: "))
	
	sum = first_number + second_number
	print(f"The sum of the number is: {sum}")

	print("do you want to continue")
	repeat = input("yes/no : ")

else: 
	print("Thank You")

	
	